### PR TITLE
Add 1-WPK day 3

### PR DESCRIPTION
**Pull request by camargodev**

### CODE REVIEW

File: hackerrank/1-week-prep/day-3/ceaser_cypher.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+ALPHABET = 'abcdefghijklmnopqrstuvwxyz'
+
+def caesarCipher(text, num_rotations):
```
Comment (jdalbosco) [inline hackerrank/1-week-prep/day-3/ceaser_cypher.py]:
```suggestion
def caesar_cipher(text, num_rotations):
```

File: hackerrank/1-week-prep/day-3/mock-test-3.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+def palindromeIndex(word):
```
Comment (jdalbosco) [inline hackerrank/1-week-prep/day-3/mock-test-3.py]:
```suggestion
def palindrome_index(word):
```

File: hackerrank/1-week-prep/day-3/tower-breaks.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+def tower_breakers(n, m):
+    if m == 1:
+        return 2
+    return 2 if n % 2 == 0 else 1
```
Comment (jdalbosco) [inline hackerrank/1-week-prep/day-3/tower-breaks.py]: I think this code would benefit from a better variable naming. 😸

File: hackerrank/1-week-prep/day-3/zig-zag-sequence.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+def find_zig_zag_sequence(a, n):
+    a.sort()
+    mid = int((n + 1)/2)-1
+    a[mid], a[n-1] = a[n-1], a[mid]
+
+    st = mid + 1
+    ed = n - 2
```
Comment (jdalbosco) [inline hackerrank/1-week-prep/day-3/zig-zag-sequence.py:7]: I do not condone this way of naming variables.

**Review by jdalbosco — CHANGES_REQUESTED** [review]

File: hackerrank/1-week-prep/day-3/zig-zag-sequence.py (hunk 4 above)
Comment (camargodev) [reply hackerrank/1-week-prep/day-3/zig-zag-sequence.py:7]: https://help.hackerrank.com/hc/en-us

File: hackerrank/1-week-prep/day-3/ceaser-cipher.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+            encrypted_text += letter
+            continue
+        is_upper = letter.isupper()
+        letter_index = ALPHABET.index(letter.lower())
+        encrypted_letter_index = (letter_index+num_rotations)%len(ALPHABET)
+        encrypted_letter = ALPHABET[encrypted_letter_index]
+        if is_upper:
+            encrypted_letter = encrypted_letter.upper()
+        encrypted_text += encrypted_letter
+    return encrypted_text
```
Comment (jdalbosco) [inline hackerrank/1-week-prep/day-3/ceaser-cipher.py:16]: _*chef's kiss*_

File: hackerrank/1-week-prep/day-3/tower-breaks.py [hunk 6]
```diff
@@ -0,0 +1,9 @@
+PLAYER_1_WINS = 1
+PLAYER_2_WINS = 2
+
+def tower_breakers(number_of_towers, tower_height):
+    has_possible_moves = tower_height > 1
+    has_even_num_of_towers = number_of_towers % 2 == 0 
+    if not has_possible_moves:
+        return PLAYER_2_WINS
+    return PLAYER_2_WINS if has_even_num_of_towers else PLAYER_1_WINS
```
Comment (jdalbosco) [inline hackerrank/1-week-prep/day-3/tower-breaks.py:9]: Looking great! 💯

**Review by jdalbosco — APPROVED** [review]
👨‍🍳

**Pull request merged**